keyboards/1upkeyboards/super16/rules.mk

```diff
@@ -62,22 +59,20 @@ BOOTLOADER = caterina
 # Build Options
 #   change yes to no to disable
 #
-BOOTMAGIC_ENABLE = lite      # Virtual DIP switch configuration(+1000)
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration(+1000)
 MOUSEKEY_ENABLE = yes       # Mouse keys(+4700)
 EXTRAKEY_ENABLE = yes       # Audio control and System control(+450)
-CONSOLE_ENABLE = yes        # Console for debug(+400)
-COMMAND_ENABLE = yes        # Commands for debug and configuration
+CONSOLE_ENABLE = no         # Console for debug(+400)
```

Comment:
We have plenty of space left on the eeprom to keep these as yes, are you encountering issues?

Reply:
Yeah, with those on I was getting errors:

```
Making 1upkeyboards/super16 with keymap default                                                        [ERRORS]

 * The firmware is too large! 30798/28672 (2126 bytes over)
```

I think this is due to the addition of the `#define RGBLIGHT_ANIMATIONS` potentially?

Reply:
Probably, you can try adding in optimization flags

```
EXTRAFLAGS += -flto
```

Reply:
Yeah, animations take up a lot of space. 

Also, disabling macros/functions may be a good idea, too.

Reply:
With those flags it's still getting really close:

```
Making 1upkeyboards/super16 with keymap default                                                        [WARNINGS]

 * The firmware size is approaching the maximum - 28190/28672 (482 bytes free)
```
 
I'm open to whatever you think is best here.

Reply:
Alright that's troubling. When rgb matrix is implemented this is going to get even bigger. Let's stick with turning them off for now.